### ISSUE TITLE
Selects that have a "blank" value does not align the dropdown icon correctly

### ISSUE THREAD

**Issue by rdy** [pivotal-cf/pivotal-ui#537]
## What version of Pivotal UI are you using?
9
## What browser and version are you using?
Chrome
## What behavior are you seeing?
Select component that have "blank" selected show the icon for the select in a weird alignment. This does not happen if you always have a valid selected value.
## What do you expect to see?
It should align the icon centered.

![screen shot 2017-10-25 at 11 24 54 am](https://user-images.githubusercontent.com/21672/32015958-76be5768-b977-11e7-9dab-16830fd0a1ac.png)

**Comment by sjolicoeur** [comment]
@rdy can you give a code example? it seems to work in the styleguide, still trying to reproduce.

**Comment by rdy** [comment]
```
<div>
    <Select name='even-numbers' defaultValue={undefined} options={['zero', 'two', 'four', 'six', 'eight']}/>
</div>
```

**Comment by sjolicoeur** [comment]
@rdy fix will be in `v11.1`
